### DISCUSSION TITLE
Request for Assistance Regarding Extension Privacy Policy URL Issue

### DISCUSSION THREAD

**Discussion by tearill** [microsoft/MicrosoftEdge-Extensions#115] [Q&A] [answered]
I am writing on behalf of BaiduAI_Plugin@baidu.com to seek your assistance regarding the recent review of our extension for publication on the Edge Web Store. The extension review has been rejected twice, citing an issue with the privacy policy URL.

The privacy policy URL we provided during the submission process is: [https://chat.baidu.com/page/privacy](https://chat.baidu.com/page/privacy)

We have verified the accessibility of the provided URL using VPN, and it is accessible from the United States region. However, we have received feedback that the URL is not resolving to a functional web page during the extension review.

Could you please provide more details about the environment and steps used during the review process that might have caused this discrepancy? Additionally, if there are specific requirements or considerations related to privacy policy URLs that we need to adhere to, we would appreciate your guidance.

We are committed to resolving this issue promptly to ensure compliance with all guidelines and requirements for publishing our extension on the Edge Web Store. Your assistance and guidance in this matter would be highly appreciated.

Thank you very much for your time and support.

**Comment by Bhuvana-Priya** [comment, marked as answer]
Hi @tearill,

Thank you for reporting this.

I have verified and could not access the privacy policy URL you have provided. It should be a generally accessible link and not through VPN. Could you please check that and re-submit your extension?
